mappings/net/minecraft/world/CommandBlockExecutor.mapping

```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/class_1918 net/minecraft/world/CommandBlockExecutor
+	COMMENT A common logic for command {@linkplain net.minecraft.block.entity.CommandBlockBlockEntity
+	COMMENT block} and {@linkplain net.minecraft.entity.vehicle.CommandBlockMinecartEntity
+	COMMENT minecart}, to reduce duplicate code.
```

Comment:
```suggestion
	COMMENT A common handler for executing and tracking the output of commands
	COMMENT run by {@linkplain net.minecraft.block.entity.CommandBlockBlockEntity
	COMMENT command blocks} and {@linkplain net.minecraft.entity.vehicle.CommandBlockMinecartEntity
	COMMENT command block minecarts}.
```

Reply:
```suggestion
	COMMENT A common logic for command-block behaviors shared by
	COMMENT {@linkplain net.minecraft.block.entity.CommandBlockBlockEntity
	COMMENT command blocks} and {@linkplain net.minecraft.entity.vehicle.CommandBlockMinecartEntity
	COMMENT command block minecarts}.
```
Imo we can generalize this to "command block behavior" than the specific `@` name, logging, redstone input/output etc.